README.md

```diff
@@ -215,6 +215,9 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which support continuously building, testing and releasing applications.*
 
 * [Bamboo ![c]](https://www.atlassian.com/software/bamboo) - Atlassian's solution with good integration of their other products. You can either apply for an open-source license or buy it.
+* [Buddy ![c]](https://buddy.works/) - Hosted CI/CD platform with on-premise option.
+* [CircleCI ![c]](https://circleci.com) - Hosted CI/CD platform with docker and OSX support. Concurrent builds rather than project based billing model.
```

Comment:
Okay, that's fine, but then we should split "OSX" into "OS X" as they do on the CircleCI website.